Frontend/src/App.js

```diff
@@ -11,9 +11,9 @@ import PrivateRoute from './components/Routing/PrivateRoute';
 import { HashRouter, Switch, Route, Redirect } from 'react-router-dom';
 
 function App() {
-  const [authenticated, setAuthenticated] = useState(false);
-  const [token, setToken] = useState(null);
-  const [isAdmin, setIsAdmin] = useState(false);
+  const [authenticated, setAuthenticated] = useState(true);
```

Comment:
This actually causes a merge conflict, so before a PR let's set these back to default values for everyone's ease